contrib/kind.sh

```diff
@@ -163,6 +163,8 @@ parse_args() {
                                                 ;;
             -ikv | --install-kubevirt)          KIND_INSTALL_KUBEVIRT=true
                                                 ;;
+            -ipam | --install-ipam-controller) KIND_INSTALL_IPAM_CONTROLLER=true
```

Comment:
we can't have this flag name. 

We have IPAM already; it is misleading. What we want to have is IPAM for virt. Maybe that name is good enough.

Reply:
nit: please also fix ident (at least looks like that)

Reply:
suggestions for new name? `ipam-ext`? `ipam-kv`?

Reply:
kubevirt-ipam ?

Reply:
short flag is `-kvipam` 
long flag is `--install-kubevirt-ipam`

Reply:
please add the above info to the PR desc
it is helpful that PR desc has a TL;DR implicit user guide style

Reply:
still missing please i think, even that marked with emoji